### ISSUE TITLE
More errors during buildout

### ISSUE THREAD

**Issue by soerendip** [senaite/senaite.lims#107]
The installation instructions seem outdated. During buildout I get these weird errors:
 https://www.senaite.com/docs/installation

```
...
zip_safe flag not set; analyzing archive contents...
Installing backports.functools-lru-cache 1.6.1
Caused installation of a distribution:
UNKNOWN 0.0.0
with a different project name.
Installing backports.functools-lru-cache 1.6.1
Caused installation of a distribution:
UNKNOWN 0.0.0
with a different version.
Path /usr/local/Plone/buildout-cache/eggs/UNKNOWN-0.0.0-py2.7.egg unexpectedly already exists.
Maybe a buildout running in parallel has added it. We will accept it.
If this contains a wrong package, please remove it yourself.
Got None.
Version and requirements information containing backports.functools-lru-cache:
While:
  Installing client1.
Error: There is a version conflict.
We already have: UNKNOWN 0.0.0
but soupsieve 1.9.5 requires 'backports.functools_lru_cache; python_version < "3"'.
```

and running it gives:

```
sudo bin/plonectl start
daemon manager not running
client1: Traceback (most recent call last):
  File "bin/plonectl", line 14, in <module>
    sys.exit(plone.recipe.unifiedinstaller.ctl.main(server='zeoserver', clients=['client1', 'client2'], location=r'/usr/local/Plone/zeocluster', binDirectory=r'/usr/local/Plone/zeocluster/bin', fileStorage=r'/usr/local/Plone/zeocluster/var/filestorage/Data.fs'))
  File "/usr/local/Plone/buildout-cache/eggs/plone.recipe.unifiedinstaller-4.3.2-py2.7.egg/plone/recipe/unifiedinstaller/ctl.py", line 120, in main
    returncode = controller.runCommand(component, command)
  File "/usr/local/Plone/buildout-cache/eggs/plone.recipe.unifiedinstaller-4.3.2-py2.7.egg/plone/recipe/unifiedinstaller/ctl.py", line 83, in runCommand
    po = subprocess.Popen(args)
  File "/usr/local/Plone/Python-2.7/lib/python2.7/subprocess.py", line 390, in __init__
    errread, errwrite)
  File "/usr/local/Plone/Python-2.7/lib/python2.7/subprocess.py", line 1024, in _execute_child
    raise child_exception
OSError: [Errno 2] No such file or directory
```

Seems the python build does not work properly. Why are you bothering creating a miniconda environment when you don't use it?

**Comment by soerendip** [comment]
On Ubuntu 18.04

**Comment by xispa** [comment]
HI @soerendip , look to https://github.com/senaite/senaite.lims/issues/106#issuecomment-556196959

**Comment by ramonski** [comment]
> Seems the python build does not work properly. Why are you bothering creating a miniconda environment when you don't use it?

Installing miniconda activates a `base` environment, which ensures that you are on Python 2.7.x per default. Therefore, the global `python` variable should refer to this:

```shell
$ which python
~/miniconda2/envs/senaite/bin/python
```

I'm closing this one, because it is a duplicate of https://github.com/senaite/senaite.lims/issues/106
